src/index.js

```diff
@@ -1,6 +1,5 @@
 /* This is the entry point of the public package interface */
 
-import '../materials/internals/reset.mod.css';
```

Comment:
Can you elaborate how we can be sure that no app-code implicitly depends on these?

Reply:
because both of those files are duplicated in the app shell and are imported there

https://github.com/commercetools/merchant-center-application-kit/blob/master/packages/application-shell/src/components/application-shell/reset.mod.css

Reply:
😄